Libraries/Components/TextInput/TextInput.js

```diff
@@ -880,7 +880,7 @@ const TextInput = createReactClass({
 
     return (
       <TouchableWithoutFeedback
-        onLayout={this._onLayout}
```

Comment:
can we do this.props.onLayout to maintain equality?

Reply:
@chirag04 - `props` is copied into a local variable:
https://github.com/facebook/react-native/blob/master/Libraries/Components/TextInput/TextInput.js#L836

This is exactly how it's done for iOS:https://github.com/facebook/react-native/blob/master/Libraries/Components/TextInput/TextInput.js#L771
https://github.com/facebook/react-native/blob/master/Libraries/Components/TextInput/TextInput.js#L822